components/ShoppableHome/ShoppableHome.module.scss

```diff
@@ -0,0 +1,169 @@
+.shoppableHome {
+    &__link {
+        cursor: default;
+    }
+
+    &__container {
+        cursor: pointer;
+        padding: spacer(base);
+        margin: spacer(xs) 0;
+        height: 300px;
+        width: 100%;
+        bottom: 0;
+        align-items: flex-end;
+        background-position: center center;
+        background-repeat: no-repeat;
+        background-size: cover;
+
+        @include breakpoint(md){
+            height: 202px;
+        }
+
+        @include breakpoint(lg){
+            margin: spacer(sm) 0;
+            height: 300px;
+        }
+
+        &:hover { 
+            background: color(contrast, white);
+            opacity: .85;
+            background-position: center center;
+            background-repeat: no-repeat;
+            background-size: cover;
+        }
+
+        &--saatva-classic {
+            @include responsive-dpi-img('home/shoppable-home', 'shop-classic-v1', jpg, false, true, false);
```

Comment:
Can we change this from being in the css, to use the Next/Image component with a type of responsive?

I really want to showcase how extremely fast nextjs can be by just using the tools in the box that they give.  Im going to do a website speedtest in the demo and I think that removing all of the extra images that have to be added for this to work, making it one big image and then letting image/next handle the multiple sizes for us will really boost the homepage speed. with much less code on our end.